openpype/hosts/standalonepublisher/plugins/publish/collect_editorial_csv.py

```diff
@@ -0,0 +1,752 @@
+from copy import copy, deepcopy
+import re
+import os
+import csv
+from pprint import pformat
+import pyblish.api
+from openpype.lib.transcoding import (
+    VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+)
+from openpype.client import get_asset_by_name
+from openpype.pipeline.create import get_subset_name
+from openpype.pipeline import (
+    publish
+)
+import clique
+
+
+class CollectEditorialCSV(
+    pyblish.api.InstancePlugin,
+    publish.ColormanagedPyblishPluginMixin
+):
+    """Collect Editorial CSV"""
+
+    order = pyblish.api.CollectorOrder + 0.002
+    label = ">>>> Collect Editorial CSV"
+    hosts = ["standalonepublisher"]
+    families = ["editorialcsv"]
+
+    # variables
+    csv_delimiter = ","
+    tags_delimiter = ";"
+    default_tags = ["review"]
+
+    # configs
+    columns_config = None
+    representation_config = None
+
+    def process(self, instance):
+        context = instance.context
+
+        passing_instance_data = {
+            _key: _value for _key, _value in instance.data.items()
+            if _key in [
+                "asset", "family", "subset", "label", "publish", "source"
+            ]
+        }
```

Comment:
This will be slow if `instance.data` has many keys.

```suggestion
        transfer_keys = [
                "asset", "family", "subset", "label", "publish", "source"
        ]
        passing_instance_data = {
            _key: instance.data[_key] for _key in transfer_keys 
            if key in instance.data
        }
```

This will likely be faster since it's less iterations (most likely; even though the collector does run early so maybe the instances don't have much data keys yet?) but mostly because it does the lookup against the `instance.data` `dict` instead of the list.

If you really prefer the opposite iteration, then it should be comparing against a predefined `set` instead of comparing at a list that gets regenerated at each iteration.

```suggestion
        transfer_keys = {
            "asset", "family", "subset", "label", "publish", "source"
        }
        passing_instance_data = {
            _key: _value for _key, _value in instance.data.items()
            if _key in transfer_keys
        }
```

Reply:
I apologize for taking your time on this. I should have mentioned it in the description that `standalonepublisher` was only quick POC - I had included it in this PR since it was related and was used as kick of for the implementation but we are not going to finalize it any farther.